src/lib/LUA/tx_devLUA.cpp

```diff
@@ -444,6 +444,10 @@ static void registerLuaParameters()
 static int event()
 {
   uint8_t rate = adjustPacketRateForBaud(config.GetRate());
+  vtxFolderDynamicName[10] = (config.GetVtxBand() + 65);
+  vtxFolderDynamicName[12] = (config.GetVtxChannel() + 48);
+  vtxFolderDynamicName[14] = (config.GetVtxChannel() + 48);
+  vtxFolderDynamicName[16] = (config.GetVtxChannel() + 80);
```

Comment:
Maybe it would be better to make create a variable for the first index.
```
  char *index = strchr(vtxFolderDynamicName, ';') - 1;
  index[0] = config.GetVtxBand() + 65;
  index[2] = config.GetVtxChannel() + 48;
  index[4] = config.GetVtxChannel() + 48;
  index[6] = config.GetVtxChannel() + 80;
```